docs/user-guide/src/bmo/firmware_components.md

```diff
@@ -0,0 +1,150 @@
+# Firmware Components
+
+Metal3 supports modifying firmware components of the hosts before provisioning
+them or after. This feature can be used to update the firmware for BIOS or BMC. The corresponding functionality in Ironic is called [Firmware
```

Comment:
"before or after provisioning" ?

Reply:
Reading now I'm a bit worried that this would give the idea that you can change the firmware component when the host is `provisioned` 🤔  (but in this case it would go to `deprovisioning` ... `preparing` `available` `provisioning` again...)

Reply:
yeah, it needs to be more clear that the node will go into deprovisioning it it's provisioned, and all the steps

Reply:
I will let
"Metal3 supports modifying firmware components of the bare metal hosts." since in ## How to change firmware components I 've explained how it works in both cases.